apps/test/unit/instructionsComponentsTest.js

```diff
@@ -10,73 +10,67 @@ describe('instructions components', () => {
 
   describe('MarkdownInstructions', function() {
     it('standard case had top padding and no left margin', function() {
-      var dom = mount(
-        <div>
-          <StatelessMarkdownInstructions
-            markdown="md"
-            markdownClassicMargins={false}
-            inTopPane={false}
-            noInstructionsWhenCollapsed={true}
-          />
-        </div>
+      const wrapper = mount(
+        <StatelessMarkdownInstructions
+          markdown="md"
+          markdownClassicMargins={false}
+          inTopPane={false}
+          noInstructionsWhenCollapsed={true}
+        />
       );
-      var element = dom.find('.instructions-markdown').first();
+      const element = wrapper.find('.instructions-markdown').first();
       expect(element.props().style.paddingTop).to.equal(19);
       expect(element.props().style.marginBottom).to.equal(35);
       expect(element.props().style.marginLeft).to.equal(undefined);
       expect(element.text()).to.equal('md');
     });
 
     it('inTopPane has no top padding', function() {
-      var dom = mount(
-        <div>
-          <StatelessMarkdownInstructions
-            markdown="md"
-            inTopPane={true}
-            noInstructionsWhenCollapsed={true}
-          />
-        </div>
+      const wrapper = mount(
+        <StatelessMarkdownInstructions
+          markdown="md"
+          inTopPane={true}
+          noInstructionsWhenCollapsed={true}
+        />
       );
-      var element = dom.find('.instructions-markdown').first();
+      const element = wrapper.find('.instructions-markdown').first();
       expect(element.props().style.paddingTop).to.equal(0);
     });
   });
 
   describe('NonMarkdownInstructions', function() {
     it('can have just instructions', function() {
-      var dom = mount(
-        <div>
-          <NonMarkdownInstructions
-            puzzleTitle="title"
-            shortInstructions="instructions"
-          />
-        </div>
+      const wrapper = mount(
+        <NonMarkdownInstructions
+          puzzleTitle="title"
+          shortInstructions="instructions"
+        />
       );
-      var elements = dom.childAt(0).children();
+      const elements = wrapper
+        .find('div')
+        .first()
+        .children();
       expect(elements.length).to.equal(2);
-      expect(elements.first())
-        .text()
-        .to.equal('title');
-      expect(elements.last())
-        .text()
-        .to.equal('instructions');
+      expect(elements.first().text()).to.equal('title');
+      expect(elements.last().text()).to.equal('instructions');
```

Comment:
We have to call `.text()` on the wrapper, not the expectation, because reconfiguredChai does not include `chai-enzyme`.